dist/tools/picotool/picotool.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+# $1 contains generated elf file - $FLASHFILE from makefile
+
+ELFFILE="$1"
+UF2FILE="${UF2FILE:-${ELFFILE%%.elf}.uf2}"
+
+udev_warning='[WARNING] Picotool udev rules are not set yet. You won`t be able to '\
+    'run picotool without sudo. See: '\
+    'https://github.com/raspberrypi/picotool/tree/master?tab=readme-ov-file#linux--macos'
```

Comment:
```suggestion
udev_warning='[WARNING] Picotool udev rules are not set yet. You won`t be able to '\
'run picotool without sudo. See: '\
'https://github.com/raspberrypi/picotool/tree/master?tab=readme-ov-file#linux--macos'
```

No indentations can be used here, otherwise the string concatenation won't work unfortunately as you can see in the log below.